huggingface/tensorflow/inference/docker/2.8/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,134 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG MMS_VERSION=1.1.8
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.9.10
+ARG OPEN_MPI_VERSION=4.0.1
+# HF ARGS
+ARG TF_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/tensorflow/r2.8_aws/cpu/2022-02-07-21-14/tensorflow_cpu-2.8.0-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
+ARG TRANSFORMERS_VERSION
+
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    LD_LIBRARY_PATH="/opt/conda/lib/:${LD_LIBRARY_PATH}:/usr/local/lib" \
+    PYTHONIOENCODING=UTF-8 \
+    LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    TEMP=/home/model-server/tmp \
+    DEBIAN_FRONTEND=noninteractive
+
+ENV PATH /opt/conda/bin:$PATH
+
+RUN apt-get update \
```

Comment:
it looks like something in this statement is returning nonzero exit code. can you debug, either by reproducing manually, or by splitting the statement into multiple run commands to identify where it is failing?

Reply:
I changed it to individual installs. But i cannot check the logs since it is "stuck". P.S. Its the same packages we use PyTorch CPU Inference container so not sure why this should fail

Reply:
On reading the logs I found out that GPU images , training and inference both, succeed the image size check. However, we do not see anything about the CPU image size check. There is a probability that CPU inference image fails the size check and exits in a way that causes the hang. I would suggest increasing the image baseline size for the CPU inference image and see if the builds succeed.

Reply:
@philschmid I think that was the reason. The builds have started succeeding. We would want to take the following steps now:
1. See what is the new size of the CPU inference images and what was the old size. If there is a drastic difference, we might want to dive deep and see what is causing the inflation. 
2. As of now, I have set the `image_size_baseline` to 6000 (that I increased from 3000). We would want to set this size based on the size of the newly built images.